### PR TITLE
ADEN-8010 Wdio Tests Refactor

### PR DESCRIPTION
**Pull request by emadry**
Part 1 of this task - refactor and stability improvement.
JW player page added

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 /* eslint-disable import/no-extraneous-dependencies */
-const networkCapture = require('./tests/common/networkCapture');
+const networkCapture = require('./tests/common/network-capture');
 const path = require('path');
 const VisualRegressionCompare = require('wdio-visual-regression-service/compare');
 const md5 = require('js-md5');
 
+
```
Comment (emadry) [inline wdio.conf.js]: To be removed

File: tests/common/helpers.js [hunk 2]
```diff
@@ -32,6 +30,15 @@ class Helpers {
 		);
 	}
 
+	waitForViewabillityCounted(timeout = timeouts.viewabillity) {
+		browser.pause(timeout);
+	}
+
+
```
Comment (dmnsobczak) [inline tests/common/helpers.js]: Enters!

File: tests/common/helpers.js [hunk 3]
```diff
@@ -99,17 +89,16 @@ class Helpers {
...
-	 * @param adSlot slot to wait for
-	 * @param timeout duration of the pause
-	 */
 	refreshPageAndWaitForSlot(adSlot, timeout = timeouts.standard) {
 		browser.refresh();
 		browser.pause(timeout);
 		browser.waitForVisible(adSlot, timeout);
 	}
 
+	waitForVideoAdToFinish(videLength) {
```
Comment (dmnsobczak) [inline tests/common/helpers.js]: Typo

File: tests/common/query-strings.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+	getSessionIdParam(parameter) {
+		return `${this.utils.sessionId}=${parameter}`;
+	}
+
+	constructInstantGlobal(wg, country = 'XX', percent = null, additional = null) {
+		return `${wg}=${this.getIGParameters(country, percent, additional)}`;
+	}
+}
+
+export default new QueryStrings();
```
Comment (dmnsobczak) [inline tests/common/query-strings.js]: Consider `export queryStrings = new QueryStrings();`

**Pull request merged**